actors/market/src/ext.rs

```diff
@@ -26,13 +26,27 @@ pub mod miner {
     use super::*;
 
     pub const CONTROL_ADDRESSES_METHOD: u64 = 2;
```

Comment:
yes i will

Reply:
@anorth didn't remove this because it's also used in `escrow_address`
https://github.com/filecoin-project/builtin-actors/blob/3d501f98071b84f7a16af12f48c1c30014fcdef4/actors/market/src/lib.rs#L1354

Reply:
we can rename this to `request_miner_addrs` is that is more suitable

Reply:
#1092